### ISSUE TITLE
Rendering in Orthogonal View Angle

### ISSUE THREAD

**Issue by Luh1124** [NVIDIAGameWorks/kaolin#803]
Hello, I am trying to render a glb file using orthogonal view angle, which involves the function generate_pinhole_rays_dir to determine the camera viewing direction. I have defined a function to calculate the viewing direction for orthogonal view angle. Can you please check if it is correct?

def generate_orthographic_rays_dir(camera, height, width, device='cuda'):
    """Generate centered grid.
    
    This is a utility function for specular reflectance with spherical gaussian.
    """
    points_camera = torch.tensor([[0,0,1], [0,0,2]], dtype=torch.float32, device=device)
    ray_dir = points_camera[0] - points_camera[1]
    ray_dir = ray_dir.expand(1, 1, 3)
    ray_org = torch.zeros_like(ray_dir)

    ray_orig, ray_dir = camera.extrinsics.inv_transform_rays(ray_org, ray_dir)

    ray_dir = ray_dir / torch.linalg.norm(ray_dir, dim=-1, keepdim=True)

    return ray_dir.expand(1, height, width, 3)

**Comment by Luh1124** [comment]
And，how can I load environment lighting for rendering? Just like in Blender.

**Comment by orperel** [comment]
Hi @Luh1124 !
With the recent release we just included a[ raygen function for ortho cameras](https://github.com/NVIDIAGameWorks/kaolin/blob/master/kaolin/render/camera/raygen.py#L172).

See usage example here:
https://github.com/NVIDIAGameWorks/kaolin/blob/master/examples/recipes/camera/camera_ray_tracing.py

The `generate_rays` api should automatically call `generate_ortho_rays` under the hood.

**Comment by shumash** [comment]
fixed; marking as closed.
